octoprint_mrbeam/static/js/app/view-models/wizard/netconnectd-wrapper.js

```diff
@@ -20,21 +20,53 @@ $(function () {
         self.onStartup = function () {
             // needs to be scrollable on touch devices
             $("#wizard_dialog .modal-body").addClass("scrollable");
+            self.onConnectionOptionSelection(
+                $("#connection-wizard__a--option-a"),
+                $("#connection-wizard--selected-option-a")
+            );
+            self.onConnectionOptionSelection(
+                $("#connection-wizard__a--option-b"),
+                $("#connection-wizard--selected-option-b")
+            );
+            self.onConnectionOptionSelection(
+                $("#connection-wizard__a--option-c"),
+                $("#connection-wizard--selected-option-c")
+            );
+            $(".connection-wizard__btn--back").click(function () {
+                self.onBackToAllOptions();
+            });
+        };
 
-            $("#wizard_welcome_wifi_config_btn").click(function () {
-                $("#wizard_welcome_wifi_configuration").show(500);
-                $("#wizard_dialog > .modal-body").animate(
-                    { scrollTop: $("#wizard_dialog > .modal-body").height() },
-                    "slow"
-                );
+        self.onConnectionOptionSelection = function (
```

Comment:
This is just adding the click listener right? Then I would rename the function I was a bit confused with this name

Reply:
Oh you are right here; I wanted to rename it because it is misleading

Reply:
done